apps/taxes/src/modules/avatax/avatax-webhook.service.ts

```diff
@@ -16,7 +17,7 @@ export class AvataxWebhookService implements ProviderWebhookService {
   client: AvataxClient;
   private logger: Logger;
 
-  constructor(config: AvataxConfig) {
+  constructor(config: AvataxConfig, private ctx: AuthData) {
```

Comment:
either
```
type CTX = { authData: AuthData }
```

or just `authData: AuthData`

once ctx grows, you will have to rewrite everything

also ctx is a bad name, it can stand for many things. In tRPC context is something defined by lib, but in custom class, its ambiguous.

Reply:
You resolved it but the change is not applied

Reply:
it is now